index.html

```diff
@@ -3555,6 +3555,21 @@ <h3>
 Signatures on <a>DID documents</a> are optional. <a>DID method</a>
 specifications SHOULD explain and specify their implementation if applicable.
         </p>
+
+        <div class="note" role="note" id="issue-container-generatedID-8">
+          <div role="heading" class="note-title marker" id="h-note-7" 
+          aria-level="4"><span>Note</span></div><p class="">
+Through the use of digital signatures a controller can assert proof of control 
```

Comment:
"proof of control" needs to be defined somewhere... it's used a number of times in the specification without really going into what "proof of control" and "proof of ownership" mean... we need to be explicit about what we mean. It is effectively "Using private cryptographic material to generate a digital signature to prove that one is in control of the private cryptographic material."